tests/framework/utils.py

```diff
@@ -680,7 +680,7 @@ def populate_data_store(test_microvm, data_store):
 
 def start_screen_process(screen_log, session_name, binary_path, binary_params):
     """Start binary process into a screen session."""
-    start_cmd = "screen -L -Logfile {logfile} " "-dmS {session} {binary} {params}"
```

Comment:
Thank you! This weird space has been bothering me so much, but I kept forgetting to actually do something about it :joy:

Reply:
yw :)!